test/NNPDE_tests.jl

```diff
@@ -446,41 +427,26 @@ inn = 18
 chain = FastChain(FastDense(1,inn,Flux.σ),
                   FastDense(inn,inn,Flux.σ),
                   FastDense(inn,1))
+initθ = Float64.(DiffEqFlux.initial_params(chain))
```

Comment:
it looks like that BFGS stock with inner convert Float32/Float64 if all in Float64 is work fine

Reply:
NN(x) + x 
convert(Float32 + Float64) -> Float64
and we get Float64 with Float32 precision

Reply:
Yeah, it's up to the user to just make sure that types match.